app/templates/results.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Extraction Results</title>
     <link href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2.19/dist/tailwind.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/inspect.js"></script>
```

Comment:
It appears that I included the `inspect.js` library in the HTML file but did not implement its functionality in the JavaScript code. I will correct this by updating the JavaScript to utilize `inspect.js` for overlaying and manipulating text on images as intended.

I will amend the PR with the necessary changes to the JavaScript code in the `results.html` file to properly use `inspect.js`. This will include initializing the library and setting up the required functionality for text overlay, dragging, and resizing if supported.